packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageBox.tests.js

```diff
@@ -80,6 +79,24 @@ QUnit.module('MessageBox', moduleConfig, () => {
 
             assert.strictEqual(this.$input.val(), emptyValue);
         });
+
+        QUnit.test('textarea should not be cleared on enter key when no text is entered', function(assert) {
+            keyboardMock(this.$input)
+                .focus()
+                .type('   ')
+                .keyUp('enter');
+
+            assert.strictEqual(this.$input.val(), '   ');
```

Comment:
I think that check value is no fully correct for our cases. We use text options, and it can not be synchronized with value. Seems, it's better to check text or both options.

Reply:
Sorry, i initially missed this comment. Do you insist we should explicitly check `text` property? I presume it's not so important while editor itself is tested in a similar way

Reply:
I understand your point and agree to omit my suggestion until issues with this approach occur